docs/source/model_design/general_reference_material/bliss/index.rst

```diff
@@ -44,7 +44,74 @@ Add an introductory paragraph here (include visual of 'What this is vs. what thi
 
 1.1 Guiding Principles
 ----------------------
-In this section, we will outline the guiding principles of BLISS.
+
+Our guiding principles for this document can be summarized in three points:  
+
+**1. This work is important**
+
+Inclusive language use is important, and we want to begin by acknowledging the 
+potential harm that using gendered language can create. ‘Sex’ and ‘gender’ are 
+not synonymous, unidimensional traits. [Bauer_2022]_ acknowledges this and explains 
+how common sex and gender proxies such as chromosomes or sex organs are imperfect 
+and not fully inclusive of lived experiences.  
+
+Using imperfect proxies can create real harm by alienating individuals, and by 
+creating invalid research. People who are not cisgendered (a diverse group including 
+intersex, nonbinary, transgender and other gender nonconforming individuals) experience 
+gendered language frequently which can be alienating and cause harm. These individuals 
+might be less likely to interact with healthcare, see themselves in research, and lose 
+opportunities as a result. In an effort to work toward institution-wide DEI goals and 
+advance our team’s research and publications, we created this guide to explore the 
+concepts of sex and gender and offer inclusive language alternatives.  
+
+One example of how proxy measures can affect the validity of research is in [Bauer_2022]_, 
+where researchers used 'women' as a proxy for people with a uterus. The researchers were 
+trying to find the incidence of uterine cancer, but by using 'women' as the denominator, 
+even though not all women have a uterus, cancer rates were thought to be lower than was true 
+in real lif. This was especially prominent in Black women who are more likely to have had 
+a hysterectomy. This example highlights the importance of using a measure that is actually 
+tied to the research question.  
+
+A second example of the potential impact on research validity is equating differences in 
+results to only sex or gender. As explained in [Ritz_and_Greaves_2022]_, both sex and gender 
+can lead to biological changes and untangling these pathways is complex or impossible. Often, 
+researchers will report findings as "sex differences" implying that they are a result of sex, 
+when the possibility that gendered behavior or norms was the cause was never explored. 
+
+Lastly, we hope to inspire change in other researchers by presenting our studies with gender 
+inclusive language and model practices we believe in. For all of these reasons, we feel this 
+guide is very important.  
+
+**2. This work is evolving**
+
+Language is constantly evolving, so there is rarely a single “right” choice and the accepted 
+choice may evolve over time. Therefore, this guide will be a living document that we will 
+regularly review and update. We will hold ourselves accountable for aligning the guide with 
```

Comment:
Do we have a date somewhere to note when we have last updated the doc? I guess it is all logged in git, but it might be good to include it in the readthedocs version explicitly.